ionic/src/app/api/api.service.ts

```diff
@@ -70,7 +70,7 @@ export class ApiService {
     // return await this.http.get<Wifi[]>(this.base_url + "setnickname?name=" + name).toPromise();
     try {
       if (localStorage.getItem('live') != undefined && JSON.parse(localStorage.getItem('live'))) { //live is not null and true
```

Comment:
replace this big if condition localStorage.getItem('live') != undefined && JSON.parse(localStorage.getItem('live'))

and add make a new function like if(isLive())
like this

Reply:
done